appbars/src/main/java/com/decathlon/vitamin/compose/appbars/bottomnavigations/VitaminBottomNavigations.kt

```diff
@@ -31,19 +64,207 @@ object VitaminBottomNavigations {
     ) {
         BottomNavigation(
             modifier = modifier,
-            backgroundColor = colors.background,
-            contentColor = colors.unSelected
+            backgroundColor = colors.backgroundColor,
+            contentColor = colors.unSelectedColor
         ) {
             actions.forEach {
-                BottomNavigationItem(
+                VitaminBottomNavigationItem(
```

Comment:
Question to avoid to create all these code in our project:

Because the contract of `BottomNavigationItem` is:

```kotlin
@Composable
fun RowScope.BottomNavigationItem(
    selected: Boolean,
    onClick: () -> Unit,
    icon: @Composable () -> Unit,
    modifier: Modifier = Modifier,
    enabled: Boolean = true,
    label: (@Composable () -> Unit)? = null,
    alwaysShowLabel: Boolean = true,
    interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
    selectedContentColor: Color = LocalContentColor.current,
    unselectedContentColor: Color = selectedContentColor.copy(alpha = ContentAlpha.medium)
): Unit
```

We can't ignore `selectedContentColor` and `unselectedContentColor` and specify the right color according to the selected state computed from our side to apply the right color in `label` and `icon` Composable callback?

```kotlin
val textColor = if (selceted) ... else ...
val iconColor = if (selected) ... else ...
BottomNavigationItem(
  label = { Text(text = label, color = textColor) },
  icon = { Icon(painter = ..., tint = iconColor) }
)
```

If it works, that would be great! :)

Reply:
Good idea !